src/plugins/data/public/ui/query_string_input/language_switcher.tsx

```diff
@@ -53,8 +53,8 @@ interface Props {
 }
 
 export function QueryLanguageSwitcher(props: Props) {
-  const opensearchDashboards = useOpenSearchDashboards();
-  const kueryQuerySyntaxDocs = opensearchDashboards.services.docLinks!.links.query.kueryQuerySyntax;
+  const docLinks = useOpenSearchDashboards().services.docLinks;
+  const kueryQuerySyntaxDocs = docLinks!.links.opensearchDashboards.dql.base;
```

Comment:
Should we take the opportunity to rename these `keury` variables now as referenced in https://github.com/opensearch-project/OpenSearch-Dashboards/issues/769?

Reply:
will update